.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!--
+Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+
+Please make sure that you are familiar with and follow the Code of Conduct for
+this project (found in the CODE_OF_CONDUCT.md file).
+
+Also, please make sure you're familiar with and follow the instructions in the
+contributing guidelines (found in the CONTRIBUTING.md file).
+
+If you're new to contributing to open source projects, you might find this free
+video course helpful: https://kcd.im/pull-request
```

Comment:
The url is not working

Reply:
Thanks! I will ask @kentcdodds if he could fix the redirect.

kcd.im/pull-request should point to:
```diff
-https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github?af=5
+https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github?af=5
```

Reply:
This has been fixed. Thanks for the tip!